src/module/item/spell/document.ts

```diff
@@ -324,6 +349,8 @@ class SpellPF2e extends ItemPF2e {
             // Add damage dice to instances
             for (const dice of damageDice) {
                 if (dice.override) continue;
+                // Skip dice that were ignored in the dialog
+                if (dice.ignored) continue;
```

Comment:
```suggestion
                if (dice.override || dice.ignored) continue;
```

Reply:
Well, this is likely to change anyways when I move all of this to the damage pipeline. Don't worry about it.